worksheet_A/TerminalHacking.cpp

```diff
@@ -4,8 +4,27 @@
 #include "stdafx.h"
 #include "WordList.h"
 
+using namespace std;
 const int wordLength = 5;
 const int numberOfWords = 15;
+int playerLives = 4; 
+std::string guessWord;
+int wordLengthLetter;
+int simScore = 0;
+
+// Checks the similarity between players guessed word and the correct secret word
+int simCheck(std::string guessWord, std::string secretWord)
```

Comment:
Probably better to spell out `similarity` here rather than shorten it to `sim` (which in games is usually short for "simulation").